### DISCUSSION TITLE
Sonoff Zigbee 3.0 USB e sensore temperatura/umidità Tuya

### DISCUSSION THREAD

**Discussion by Pietrogramma** [indomus/forum#2059] [ZigBee, Z-Wave, Thread, Matter]
Ciao a tutti, mi sono avvicinato da poco al mondo di Home Assisntant Supervised e sto "studiando" per cercare di capirne qualcosa.
Un paio di settimane fa ho acquistato la chiavetta Sonoff ZigBee 3.0 Usb che ho collegato al mio Raspi4 4GB sul quale gira Home Assistant ed ho installato l'integrazione Zigbee Home Automation; oggi mi è arrivato dalla Cina un sensore Temperatura e Umidità della Tuya (edit: no links) e una volta acceso mi è stato riconosciuto subito da Zigbee Home Automation, ma mi da solo due entità nascoste e nessuna entità della temperatura e della Umidità.
Ho letto che questi sensori necessiitano di un gateway della Tuya, è vero?
Se si c'è un rimedio per utilizzare il sensore con HA pur non avendo questo gateway?
Ciao e grazie

**Comment by Scialla** [comment]
Mi sa che, al momento, non è compatibile con ZHA quel modello (ZTH02)
puoi vedere la lista della compatibilità [qui](https://zigbee.blakadder.com/zha.html)

Da quel che vedo non è compatibile neanche con Z2M....

**Reply by Pietrogramma** [reply]
Grazie mille per la risposta Scialla, quindi mi devo rassegnare all'idea di non potere utilizzare questo sensore? 
leggevo da qualche parte che si poteva integrare tramite assistente google e/o tramite l'integrazione Tuya local, ti risulta?

**Reply by Scialla** [reply]
dipende quali entità espone integrandolo con Tuya Local; visto che lo hai già, prova. Poi sulla pagina che ti ho linkato c'è la procedura per richiedere l'aggiunta ai dispositivi compatibili, prova a guardare anche lì
